user-docs/latest/content-manager/configuring-view-of-content-type.md

```diff
@@ -38,4 +38,9 @@ Relational fields can also be displayed in the list view. There are however some
 - Only first-level fields can be displayed (i.e. fields from the relation of a relation can't be displayed).
 - If the displayed field contains more than one value, not all its values will be displayed, but a counter indicating the number of values. You can hover this counter to see a tooltip indicating the first 10 values of the relational field.
 
+Note also that relational fields have a couple limitations when it comes to sorting options:
```

Comment:
I would also add that you can't "double" sort.
Like first by "price" and inside that sort, by "name"
I don't know if it's worth it to mention it.